tiling.py

```diff
@@ -368,6 +486,177 @@ def fundamental_domain_transforms(self):
     def center(self):
         return self._center
 
+    @property
+    def diameter(self):
+        return self._diameter
+
+    @property
+    def radius(self):
+        return self._radius
+
+    @property
+    def group(self):
+        return self._group
+
+
+
+class vHexKernel(object):
```

Comment:
Same comment as above with `vSquareKernel`. This is not maintainable at all and hard to follow.